docs/content/guides/getting-started/setting-options.md

```diff
@@ -369,6 +431,32 @@ The [`cells`](@/api/options.md#cells) option is a function invoked before Handso
       }
     });
     ```
+:::
+
+::: only-for react
```

Comment:
In the React flavor of this "Setting row options", "Setting column options", "Setting cell options" guide, you skipped the points 1-2-3 that were present in the JavaScript part. 

Please make it analogous and also remove the points 1-2-3 from the JavaScript part.

Reply:
updated

Reply:
Are you sure? I don't see that change. In fact, maybe @kirszenbaum can take over that in case he wants to have a different opinion.

Reply:
It was there all the time in https://github.com/handsontable/handsontable/pull/9787/commits/6522261734ccc371768bf43ff884c2b4d98be1d6